device-types/Albis-Elcon/BIG4862.yaml

```diff
@@ -0,0 +1,62 @@
+---
+manufacturer: Albis-Elcon
+model: BIG4862
+slug: big4862
```

Comment:
please append `albis-elcon-` to the slug

Reply:
done